`azurerm_trusted_signing_account` - update API version & document (#31714)
terraform-providers/terraform-provider-azurerm: website/docs/r/trusted_signing_account.html.markdown

layout: "azurerm"
page_title: "Azure Resource Manager: azurerm_trusted_signing_account"
description: |-
  Manages a Trusted Signing Account (Artifact Signing Account).
---

# azurerm_trusted_signing_account

Manages a Trusted Signing Account (Artifact Signing Account).

> ~> **Note:** The resource provider is also named [Artifact Signing](https://learn.microsoft.com/en-us/azure/artifact-signing/overview).

Reviewer: Clarify 'Artifact Signing' is the rebranded name. Also do not use locale specific MS learn link.

```suggestion
~> **Note:** The resource provider has been rebranded to [Artifact Signing](https://learn.microsoft.com/azure/artifact-signing/overview).
```

Please apply equivalent changes to the data source doc

* `name` - (Required) The name which should be used for this Trusted Signing Account. Changing this forces a new Trusted Signing Account to be created.

* `resource_group_name` - (Required) The name of the Resource Group where the Trusted Signing Account should exist. Changing this forces a new Trusted Signing Account to be created.

> * `location` - (Required) The Azure Region where the Trusted Signing Account should exist. Changing this forces a new Trusted Signing Account to be created.

Reviewer: Please replace the term 'Trusted Signing' into 'Artifact Signing' everywhere in this doc, and also in the data source.
